tests/unit/test_UIARemoteOps/test_highLevel/test_iterable.py

```diff
@@ -0,0 +1,82 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2023 NV Access Limited
+
+"""
+High-level UIA remote ops Unit tests for writing iterable functions that can yield values.
+"""
+
+from unittest import TestCase
+from UIAHandler._remoteOps import operation
+from UIAHandler._remoteOps import remoteAPI
+
+
+class Test_iterable(TestCase):
+
+	def test_iterableFunction(self):
+		op = operation.Operation(localMode=True)
+
+		@op.buildIterableFunction
+		def code(ra: remoteAPI.RemoteAPI):
+			i = ra.newInt(0)
+			with ra.whileBlock(lambda: i < 4):
+				ra.Yield(i)
+				i += 1
+
+		results = []
+		for i in op.iterExecute():
+			results.append(i)
+		self.assertEqual(results, [0, 1, 2, 3])
+
+	def test_long_iterableFunction(self):
```

Comment:
could you explain this test case more in comments? I'm not sure I understand the results here